README.md

```diff
@@ -115,22 +120,26 @@ Usage
 - **database** as *string*, **required**
 
   SQLite database path.
+  
+- **updatable** as *boolean*, optional, default *true*
 
-- **truncatable** as *boolean*, optional
+  This option allow or disallow write operations on SQLite database file.
+    
+- **truncatable** as *boolean*, optional, default *false*
 
   Allows foreign tables to be truncated using the `TRUNCATE` command.
   
-- **keep_connections** as *boolean*, optional
+- **keep_connections** as *boolean*, optional, default *false*
```

Comment:
Default value of keep_connections option is true, not false.

Reply:
Corrected.

Reply:
Thanks for fixing. I confirmed.